hadoop-hdds/client/src/test/java/org/apache/hadoop/hdds/scm/storage/TestChunkInputStream.java

```diff
@@ -147,14 +145,6 @@ public void testPartialChunkRead() throws Exception {
   public void testSeek() throws Exception {
     seekAndVerify(0);
 
-    try {
```

Comment:
We have just removed the EOF check on seek. Perhaps rather than removing it, we should ensure the EOF is still thrown at CHUNK_SIZE + 1?